### ISSUE TITLE
Models and data sets

### ISSUE THREAD

**Issue by DongYan1121** [ErwinSCat/VeriFL#1]
Hello, I would like to ask you what is the model and data set of VeriFl, or do you have no model and data set to add by yourself

**Comment by ErwinSCat** [comment]
> Hello, I would like to ask you what is the model and data set of VeriFl, or do you have no model and data set to add by yourself

Our paper focuses on the (verifiable) aggregation step of FL (like Google's CCS'17 paper), and an end-to-end performance evaluation of FL is out of the scope of this paper. So, we do NOT simulate any model training in this repo. 

However, it is supposed that our aggregation method can be plugged into an FL framework that uses the standard **fixed-point encoding** of model parameters and the classical FL paradigm. To test your model and dataset with VeriFL, you may need to implement a wrapper code that realizes this fixed-point encoding and the FL procedure.

**Comment by DongYan1121** [comment]
非常感谢您的解答，我还有一个问题，那您这个有实现验证的部分吗

**Comment by DongYan1121** [comment]
Thank you very much for your answer. I have one more question，If you implement a verifiable aggregation phase, do you implement a validation phase

**Comment by ErwinSCat** [comment]
> Thank you very much for your answer. I have one more question，If you implement a verifiable aggregation phase, do you implement a validation phase

Yes. We have simulated all behaviors of each client in [this cpp](https://github.com/ErwinSCat/VeriFL/blob/main/Client/app/src/main/cpp/com_va_client_NativeClient.cpp). Those function names are consistent with the phase names in our paper. The server's code can be found in [this cpp](https://github.com/ErwinSCat/VeriFL/blob/main/Server/src/cpp/com_va_server_NativeServer.cpp).

**Comment by DongYan1121** [comment]
Hello, can I add your wechat or other contact information?

**Comment by ErwinSCat** [comment]
> Hello, can I add your wechat or other contact information?

Please try the email in my paper.
